src/main/java/com/three_tech_solutions/slot_app/controllers/implementations/PriceControllerImpl.java

```diff
@@ -0,0 +1,22 @@
+package com.three_tech_solutions.slot_app.controllers.implementations;
+
+import com.three_tech_solutions.slot_app.controllers.interfaces.PriceController;
+import com.three_tech_solutions.slot_app.controllers.requests.PriceUpdateRequest;
+import com.three_tech_solutions.slot_app.controllers.responses.PriceUpdateResponse;
+import com.three_tech_solutions.slot_app.services.interfaces.PriceService;
+import lombok.AllArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+
+@AllArgsConstructor
+@RestController
+public class PriceControllerImpl implements PriceController {
+    @Autowired
```

Comment:
El `@Autowired` permite realizar la inyeccion de dependencias de manera automatica, sin embargo no nos hace falta ya que estas haciendo la inyeccion de dependencias mediante el constructor con el `@AllArgsConstructor`


```suggestion
    private final PriceService priceService;
```